python/ray/autoscaler/node_provider.py

```diff
@@ -110,8 +110,11 @@ def find_node_id():
         return find_node_id()
 
     def create_node(self, node_config: Dict[str, Any], tags: Dict[str, str],
-                    count: int) -> None:
-        """Creates a number of nodes within the namespace."""
+                    count: int) -> Any:
+        """Creates a number of nodes within the namespace.
+
+        Optionally returns information about the created nodes.
+        """
```

Comment:
Can we specify the format of the metadata? E.g., can it always be Dict[NodeID, Any]?